src/Utilities.js

```diff
@@ -64,3 +66,23 @@ export const getLookupConfigItems = (propertyTemplate) => {
 
   return templateConfigItems
 }
+
+/**
+ * Loads N3 into a dataset.
+ * @param {string} data that is the N3
+ * @return {Promise<rdf.Dataset>} a promise that resolves to the loaded dataset
+ */
+export const loadN3 = data => new Promise((resolve) => {
```

Comment:
I wonder if improving on this name would be useful.  It *seems* like it is expecting ttl as a string to be passed in, and what comes out is an rdf-ext.Dataset object.  So "loadN3" isn't terribly obvious to me.  `rdfDatasetFromTtl` ??  "N3" is both an rdf format and the name of a package, which is even more confusing.